apps/indexer/lib/indexer/buffered_task.ex

```diff
@@ -185,6 +185,11 @@ defmodule Indexer.BufferedTask do
     GenServer.call(server, :debug_count)
   end
 
+  @doc false
+  def get_state(server, val) do
```

Comment:
Where is it used?

Reply:
I started adding new metrics to list common config values of buffered task implementations / fetchers (i.e. emit concurrency + batch size as metric values). I decided not to implement new telemetry events & the functionality to periodically emit them due to the time spent already on this feature. It's something to be added later which shouldn't take too much effort.